code/modules/surgery/surgery_helpers.dm

```diff
@@ -11,6 +11,17 @@
 	else
 		return 0.5
 
+/proc/get_organmanip_location_isopened(mob/living/carbon/located_mob, location)
+	for(var/datum/surgery/procedure in located_mob.surgeries)
```

Comment:
if we are guaranteed to not have any nulls or things not of the `/datum/surgery` type, use `as anything` like this

```suggestion
	for(var/datum/surgery/procedure as anything in located_mob.surgeries)
```